README.md

```diff
@@ -47,11 +47,29 @@ the README, using the Solr setup from `nested-template/conf/`. This boils down t
 ```shell
```

Comment:
The branch is named `nested-solr-indexing'. Also I misread the readme and thought that I had to start solr from [ds-solr](https://github.com/kb-dk/ds-solr/). However, thats not the case. Maybe specify that a more thorough documentation can be found there, but these three lines should do the trick in ds-present?

Reply:
Whoops, leftover from old instructions.